js/commonjs/rewrite_tests_for_commonjs.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * @fileoverview Utility to translate test files to CommonJS imports.
```

Comment:
I'm wondering if this translation step is needed at all. Closure already has a [bootstrap script](https://github.com/google/closure-library/blob/master/closure/goog/bootstrap/nodejs.js) that effectively converts `goog.require` to CommonJS `require`. All you need is a deps.js file to do the translation from namspace to src file. The bootstrap script linked tells the whole story and gives examples.